### ISSUE TITLE
Separate decoration settings

### ISSUE THREAD

**Issue by danielbayley** [steelbrain/linter-ui-default#227]
For example, I would like to keep the gutter highlights, but not the underlines…

However there is only one setting to enable both/neither. I think they should be separate settings.

**Comment by jonathandelgado** [comment]
I agree and I am the exact opposite, I like the underlines because I know where to mouseover for the tooltip, but I don't need gutters. Also, it appears tooltips are disabled if `Show Decorations` is disabled, even if `Show Tooltip` is enabled.

I think the best scenario would be a separate config option for each:
- Gutter
- Underlines
- Tooltip
- Panel

Which is somewhat similar to what linter had previously.

**Comment by t9md** [comment]
I want disable gutter only.
I want underline, but don't need gutter.
And if I can disable gutter decoration separately.
Enabling gutter decoration introduce side-effect which is editor's text side-shifted-at-startup because linter-ui's gutter is inserted after initial rendering.

Here is the GIF, note text side-shifted after syntax-highlighted.
If I disabled `Show Decorations` decoration, this side-shift is not happen.

![linter-guter-side-shift](https://cloud.githubusercontent.com/assets/155205/26257003/dbd67874-3cf9-11e7-865a-6ef7c0d6a86d.gif)

**Comment by kaleidoc0de** [comment]
I disabled decorations and solely using the minimap plugin until an option for either underlines or gutter lands
